Doc/library/ctypes.rst

```diff
@@ -1799,10 +1799,31 @@ different ways, depending on the type and number of the parameters in the call:
    integer. *name* is name of the COM method. *iid* is an optional pointer to
    the interface identifier which is used in extended error reporting.
 
+   If *iid* is not specified, a :exc:`WindowsError` is raised if the COM method
+   call fails. If *iid* is specified, a :exc:`.COMError` is raised instead.
+
    COM methods use a special calling convention: They require a pointer to
    the COM interface as first argument, in addition to those parameters that
    are specified in the :attr:`!argtypes` tuple.
 
+
+.. exception:: COMError(hresult, text, details)
```

Comment:
Should it be placed here or somewhere else? because it cuts the flow of the reading where *paramflags* is documented afterwards. Maybe we can put it *before* the functions?

Reply:
I was also unsure whether this was the right place to put it.

On the other hand, placing it above `prototype` or above `FOOFUNCTYPE` might seem abrupt.

Might it be appropriate to create the `Exceptions` section, including `ArgumentError`?
However, in this scope, I think making such a change that involves the "Foreign functions" section is excessive.

Reply:
Well.. honestly I think it makes sense to put it in an exception section. I think it could be fine for this small docs change (I mean, it's for our own convenience). We can ask @hugovk as a docs expert.

Reply:
Yes, we often have an exceptions section further down, see for example:

https://docs.python.org/3/library/json.html#exceptions

Reply:
If the community agrees, creating an "Exceptions" section is no problem at all.

For now, I think I'll try placing it at the end of the document, after the "Arrays and pointers" section.